build.sbt

```diff
@@ -31,7 +31,7 @@ val scalaJsDomV      = "2.8.0"
 val scalaJsReactV    = "3.0.0-beta6"
 val utestV           = "0.8.4"
 
-ThisBuild / crossScalaVersions := Seq("3.6.2")
+ThisBuild / crossScalaVersions := Seq("3.5.2")
```

Comment:
Shouldn't this be on 3.3 LTS? 

> Scala 3.4.0 code can use dependencies compiled with Scala 3.3.x, but not the other way around. That means that if you are a library author, you should stay on the LTS line.

Reply:
That's a good point. I've been evaluating pros and cons of staying on LTS.

I agree that libraries in general should stay on LTS versions. However, taking into account that only we seem to be using our libraries (if someone else is, this is the time to chime in), these libraries are not officially released (major version number is 0) and keeping up with Scala versions as they come out allow us to incrementally and gradually adapt them when needed (instead of requiring a big jump from LTS to next LTS), I think it's better to stay up to date with Scala.

At lease until we make an official 1.0.0 public release of any of them.